content/source/docs/enterprise/api/state-versions.html.md

```diff
@@ -0,0 +1,168 @@
+
+This POST endpoint requires a JSON object with the following properties as a request payload.
+
+Properties without a default value are required.
+
+
+Key path                    | Type    | Default   | Description
+----------------------------|---------|-----------|------------
+`data.type`                 | string  |           | Must be `"state-versions"`.
+`data.attributes.serial`    | integer |           | The serial of the state version
+`data.attributes.md5`       | string  |           | An MD5 hash of the raw state version
+`data.attributes.lineage`   | string  | (nothing) | Lineage of the state version
+`data.attributes.state`     | string  |           | Base64 encoded raw state file
+
+### Sample Payload
+
+```json
+{
+  "data": {
+    "type":"state-versions",
+    "attributes": {
+      "serial": 1,
+      "md5": "871d1b4a-e579-fb7c-ffdb-f0c858a647a7",
+      "lineage": "871d1b4a-e579-fb7c-ffdb-f0c858a647a7",
+      "state": "ewogICAgInZlcnNpb24iOiAzLAogICAgInRlcnJhZm9ybV92ZXJzaW9uIjog\nIjAuMTEuNyIsCiAgICAic2VyaWFsIjogMSwKICAgICJsaW5lYWdlIjogIjg3\nMWQxYjRhLWU1NzktZmI3Yy1mZmRiLWYwYzg1OGE2NDdhNyIsCiAgICAibW9k\ndWxlcyI6IFsKICAgICAgICB7CiAgICAgICAgICAgICJwYXRoIjogWwogICAg\nICAgICAgICAgICAgInJvb3QiCiAgICAgICAgICAgIF0sCiAgICAgICAgICAg\nICJvdXRwdXRzIjogewogICAgICAgICAgICAgICAgInJhbmRvbSI6IHsKICAg\nICAgICAgICAgICAgICAgICAic2Vuc2l0aXZlIjogZmFsc2UsCiAgICAgICAg\nICAgICAgICAgICAgInR5cGUiOiAic3RyaW5nIiwKICAgICAgICAgICAgICAg\nICAgICAidmFsdWUiOiAiNDI2ZmNjZTQyYTNiMTM0MCIKICAgICAgICAgICAg\nICAgIH0KICAgICAgICAgICAgfSwKICAgICAgICAgICAgInJlc291cmNlcyI6\nIHsKICAgICAgICAgICAgICAgICJyYW5kb21faWQucmFuZG9tIjogewogICAg\nICAgICAgICAgICAgICAgICJ0eXBlIjogInJhbmRvbV9pZCIsCiAgICAgICAg\nICAgICAgICAgICAgImRlcGVuZHNfb24iOiBbXSwKICAgICAgICAgICAgICAg\nICAgICAicHJpbWFyeSI6IHsKICAgICAgICAgICAgICAgICAgICAgICAgImlk\nIjogIlFtX001Q283RTBBIiwKICAgICAgICAgICAgICAgICAgICAgICAgImF0\ndHJpYnV0ZXMiOiB7CiAgICAgICAgICAgICAgICAgICAgICAgICAgICAiYjY0\nIjogIlFtX001Q283RTBBIiwKICAgICAgICAgICAgICAgICAgICAgICAgICAg\nICJiNjRfc3RkIjogIlFtL001Q283RTBBPSIsCiAgICAgICAgICAgICAgICAg\nICAgICAgICAgICAiYjY0X3VybCI6ICJRbV9NNUNvN0UwQSIsCiAgICAgICAg\nICAgICAgICAgICAgICAgICAgICAiYnl0ZV9sZW5ndGgiOiAiOCIsCiAgICAg\nICAgICAgICAgICAgICAgICAgICAgICAiZGVjIjogIjQ3ODcyNzAyMDkyNTEy\nNTEwMDgiLAogICAgICAgICAgICAgICAgICAgICAgICAgICAgImhleCI6ICI0\nMjZmY2NlNDJhM2IxMzQwIiwKICAgICAgICAgICAgICAgICAgICAgICAgICAg\nICJpZCI6ICJRbV9NNUNvN0UwQSIsCiAgICAgICAgICAgICAgICAgICAgICAg\nICAgICAia2VlcGVycy4lIjogIjEiLAogICAgICAgICAgICAgICAgICAgICAg\nICAgICAgImtlZXBlcnMudXVpZCI6ICI0OGIwOGExOS02ODA1LTIxOTctYzMw\nYS04ZmM4ZjVmOWM2MTEiCiAgICAgICAgICAgICAgICAgICAgICAgIH0sCiAg\nICAgICAgICAgICAgICAgICAgICAgICJtZXRhIjoge30sCiAgICAgICAgICAg\nICAgICAgICAgICAgICJ0YWludGVkIjogZmFsc2UKICAgICAgICAgICAgICAg\nICAgICB9LAogICAgICAgICAgICAgICAgICAgICJkZXBvc2VkIjogW10sCiAg\nICAgICAgICAgICAgICAgICAgInByb3ZpZGVyIjogInByb3ZpZGVyLnJhbmRv\nbSIKICAgICAgICAgICAgICAgIH0KICAgICAgICAgICAgfSwKICAgICAgICAg\nICAgImRlcGVuZHNfb24iOiBbXQogICAgICAgIH0KICAgIF0KfQo=\n"
```

Comment:
Wonder if we should put ... in here to shorten it.

Reply:
👍